model/lifecycle.go

```diff
@@ -756,6 +756,7 @@ func CreateTasksFromGroup(in BuildVariantTaskUnit, proj *Project) []BuildVariant
 			GroupName:        in.Name,
 			Patchable:        in.Patchable,
 			PatchOnly:        in.PatchOnly,
+			Disable:          in.Disable,
```

Comment:
Did you create a new ticket for fixing CreateTasksFromGroup?

Reply:
yes! 
https://jira.mongodb.org/browse/EVG-15837